### ISSUE TITLE
Add ability to scan QR code for sending

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-android#202]
## The Issue
When sending funds, especially on Mobile, it's useful to have the ability to scan a QR code. 

## System Configuration
- Device:
- Android version:

**Comment by toktoakhil** [comment]
Did you dropped this

**Comment by akinwale** [comment]
No, we didn't. It's not a high priority at the moment because our main focus for the app is content and discovery. We're open to contributors working on this issue, though.

**Comment by soup-jingle** [comment]
I can do this. I have it working on my device.

**Comment by kekkyojin** [comment]
Try creating a pull request, @soup-jingle
